modules/KoreBuild.Tasks/CheckPackageReferences.cs

```diff
@@ -35,14 +35,20 @@ public class CheckPackageReferences : Microsoft.Build.Utilities.Task
 
         public override bool Execute()
         {
+            if (Projects == null || Projects.Length == 0)
```

Comment:
Should we maybe do a `Log.LogVerbose` or something here? Just something to say that it succeeded because it didn't have any projects?

Reply:
updated